sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/rest_client.hpp

```diff
@@ -627,6 +627,22 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
        */
       DateTime LastModified;
     };
+    /**
+     * @brief Valid value is backup.
+     */
+    class ShareTokenIntent final {
+    public:
+      ShareTokenIntent() = default;
+      explicit ShareTokenIntent(std::string value) : m_value(std::move(value)) {}
+      bool operator==(const ShareTokenIntent& other) const { return m_value == other.m_value; }
+      bool operator!=(const ShareTokenIntent& other) const { return !(*this == other); }
+      const std::string& ToString() const { return m_value; }
+      AZ_STORAGE_FILES_SHARES_DLLEXPORT const static ShareTokenIntent Backup;
+      AZ_STORAGE_FILES_SHARES_DLLEXPORT const static ShareTokenIntent None;
```

Comment:
We should remove this enumerator.

Reply:
There's only one value for this enum, `Backup`.

Check out this link for how to do it

https://github.com/Azure/azure-sdk-for-cpp/blob/30afd1eb89f586a69f448495965311278927de55/sdk/storage/azure-storage-blobs/swagger/README.md?plain=1#L325